src/14.1/com/intellij/plugins/haxe/ide/structure/HaxeStructureViewModel.java

```diff
@@ -74,11 +74,7 @@ public boolean isVisible(TreeElement treeNode) {
       if (!(treeNode instanceof HaxeStructureViewElement)) return true;
       final PsiElement element = ((HaxeStructureViewElement)treeNode).getRealElement();
 
-      if (HaxeComponentType.typeOf(element) == HaxeComponentType.FIELD) {
-        return false;
-      }
-
-      return true;
+      return HaxeComponentType.typeOf(element) != HaxeComponentType.FIELD;
```

Comment:
I tend to leave the old code in place for these types of changes.  I write them as separate returns when I need to break in the debugger on a certain return value, because you can't set a breakpoint on the return after the expression has been evaluated.  The other pattern that I use is 
```
ret = expr; 
return ret;
```
My thinking for leaving them is that if I needed the returns separate for debugging in the past, then I'll probably need it again, and it doesn't make the code any harder to understand.  It's just a style thing, though.